components/rangeslider/rangesliders/twoHandlesSliders/twoHandlesRangeSliderBase.js

```diff
@@ -238,6 +241,35 @@ export default class TwoHandlesRangeSliderBase extends RangeSliderBase {
         document.addEventListener('mouseup', this.onMouseUp);
     }
 
+    /**
+     * Executed on touchdown. Sets the handle to the touched coordinates and attaches event listeners to the document
+     * @param {TouchEvent} e
+     */
+    onTouchDown(e) {
+        if (e.touches.length > 1) return;
+
+        const percent = this.getHandlePercent(e.touches[0]);
+
+        const currentActiveHandle = this.handle[this.activeHandle];
+        if (currentActiveHandle) this.setInactiveHandle(currentActiveHandle);
+
+        const targetClassList = e.target?.classList;
+        if (targetClassList && (targetClassList.contains('guic-horizontal-rangeslider-handle') ||
+            targetClassList.contains('guic-vertical-rangeslider-handle')) ) {
+            targetClassList.contains('handle-1') ? this.activeHandle = 1 : this.activeHandle = 0;
+        } else {
+            this.activeHandle = this.getClosestHandleToMousePosition(percent);
+        }

```

Comment:
Can't we directly check if the classList contains 'handle-1' and omit the rest checks?Why do we want to check for the horizontal or vertical rangeslider handle? Is it possible another element to have the class 'handle-1'?

Also as we have the refs to the handles, can't we simply check by ref the target?
For example:
```
if(e.target === this.handle[0]) this.activeHandle = 0;
else if(e.target === this.handle[1]) this.activeHandle = 1;
else this.activeHandle = this.getClosestHandleToMousePosition(percent);
```